modules/augment/misc/python/test/test_augmenter_images.py

```diff
@@ -0,0 +1,111 @@
+        aug.add(t=cv.augment_Rotate(minAngle=0, maxAngle=180), prob=0.3)
+        aug.add(t=cv.augment_Resize(size=(1200, 900)), prob=0.4)
+
+        imgs = []
+        masks = []
+
+        for i in range(numberOfImages):
+            widthOfImages = np.random.randint(low=MIN_IMAGE_DIM_SIZE, high=MAX_IMAGE_DIM_SIZE)
+            heightOfImages = np.random.randint(low=MIN_IMAGE_DIM_SIZE, high=MAX_IMAGE_DIM_SIZE)
+
+            img = np.random.rand(heightOfImages, widthOfImages)
+            imgs.append(img)
+            mask = np.random.rand(heightOfImages, widthOfImages)
+            masks.append(mask)
+
+        aug.applyImagesWithMasks(imgs, masks)
+
+    def test_augmenter_images_with_points(self):
+        numberOfImages = np.random.randint(low=MIN_NUMBER_OF_TESTS, high=MAX_NUMBER_OF_TESTS)
+
+        aug = cv.augment_Augmenter()
+        aug.add(t=cv.augment_FlipHorizontal(), prob=0.7)
+        aug.add(t=cv.augment_FlipVertical(), prob=0.5)
+        aug.add(t=cv.augment_GaussianBlur(kernelSize=5, sigma=12), prob=0.7)
+        aug.add(t=cv.augment_Rotate(minAngle=0, maxAngle=180), prob=0.3)
+        aug.add(t=cv.augment_Resize(size=(1200, 900)), prob=0.4)
+
+        imgs = []
+        pointsArr = []
+
+        for i in range(numberOfImages):
+            widthOfImages = np.random.randint(low=MIN_IMAGE_DIM_SIZE, high=MAX_IMAGE_DIM_SIZE)
+            heightOfImages = np.random.randint(low=MIN_IMAGE_DIM_SIZE, high=MAX_IMAGE_DIM_SIZE)
+            numberOfPoints = np.random.randint(low=MIN_NUMBER_OF_GROUND_TRUTH_DATA, high=MAX_NUMBER_OF_GROUND_TRUTH_DATA)
+
+            img = np.random.rand(heightOfImages, widthOfImages)
+            imgs.append(img)
+            points = np.random.rand(numberOfPoints, 2)
+            pointsArr.append(points)
+
+        aug.applyImagesWithPoints(imgs, pointsArr)
+
+    def test_augmenter_images_with_rectangles(self):
+        numberOfImages = np.random.randint(low=MIN_NUMBER_OF_TESTS, high=MAX_NUMBER_OF_TESTS)
+
+        aug = cv.augment_Augmenter()
+        aug.add(t=cv.augment_FlipHorizontal(), prob=0.7)
+        aug.add(t=cv.augment_FlipVertical(), prob=0.5)
+        aug.add(t=cv.augment_GaussianBlur(kernelSize=5, sigma=12), prob=0.7)
+        aug.add(t=cv.augment_Rotate(minAngle=0, maxAngle=180), prob=0.3)
+        aug.add(t=cv.augment_Resize(size=(1200, 900)), prob=0.4)
+
+        imgs = []
+        rectsArr = []
+
+        for i in range(numberOfImages):
+            widthOfImages = np.random.randint(low=MIN_IMAGE_DIM_SIZE, high=MAX_IMAGE_DIM_SIZE)
+            heightOfImages = np.random.randint(low=MIN_IMAGE_DIM_SIZE, high=MAX_IMAGE_DIM_SIZE)
+            numberOfRects = np.random.randint(low=MIN_NUMBER_OF_GROUND_TRUTH_DATA, high=MAX_NUMBER_OF_GROUND_TRUTH_DATA)
+
+            img = np.random.rand(heightOfImages, widthOfImages)
+            imgs.append(img)
+            rects = np.random.rand(numberOfRects, 4)
+            rectsArr.append(rects)
```

Comment:
We need to specify which actual types of images and rectangles are allowed. So you need to show that images must be numpy arrays and rectangles are lists or tuples or numpy arrays.

Reply:
How can I make this more clear? should I write that in comments?